wurst/systems/spawns/HostileNetBehavior.wurst

```diff
@@ -0,0 +1,27 @@
+package HostileNetBehavior
+import ClosureTimers
+import RegisterEvents
+import ID
+
+constant let MOVE_ORDER = String2OrderIdBJ("move")
```

Comment:
You could also use `Orders.smart` from package `Orders`

Reply:
/** This is a point or object targeted order that is like a right click. */

I don't really get it, what's the purpose?

Reply:
Or just `Orders.move`

Reply:
Oh yeah I could use that